dashboard/app/controllers/registrations_controller.rb

```diff
@@ -32,10 +32,10 @@ def update
             redirect_to after_update_path_for(@user)
           end
         end
-        format.json { head :no_content }
+        format.any { head :no_content }
```

Comment:
this will break the edit account page because it's expecting two different types of requests -- json and regular html post -- and wants to handle them differently.

(the two different callers are the edit account page and the age interstitial)

Why can't we just pass the accept header through? These are also all posts so are uncacheable.